pkg/webhook/channel_test.go

```diff
@@ -1,77 +0,0 @@
-/*
-Copyright 2018 The Knative Authors. All Rights Reserved.
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package webhook
-
-import (
-	"testing"
-)
-
-// TODO(n3wscott): these tests will go away when we move to new type of webhooks.
-
-func TestNewChannelNSBus(t *testing.T) {
```

Comment:
Did these tests (and the tests for the other types) all make it over to the apis packages?

Reply:
I did move them all as I was writing the other validation parts

Reply:
I can confirm. I may have pushed for some additional tests, and @n3wscott said he'd add them after this PR.